packages/go_router/lib/src/builder.dart

```diff
@@ -98,7 +98,8 @@ class RouteBuilder {
   Widget build(
     BuildContext context,
     RouteMatchList matchList,
-    bool routerNeglect,
+    bool routerNeglect, // TODO(tolo): This parameter is not used and should be
```

Comment:
Can you add a doc somewhere, probably in configuration.md?

Reply:
I meant to leave the comment in GoRouter, but yeah just document on what this routerNeglect now does (basically turn it in to a single page app without history)